inst/cpp/sim.cpp

```diff
@@ -123,7 +123,7 @@ List sim_wrapper_cpp (NumericVector A, List design, List par, NumericVector iov_
       ode_out tmp = sim_cpp(Aupd, t_start, t_start, step_size);
       // insert time-dependent covariates scale
       // insert scale definition for observation
-      // insert saving observations to obs object(s)
+      // insert saving initial observations to obs object(s)
```

Comment:
in sim.cpp template, need to split this up in generated code for the initial row (to initialize the variables) of output, and the generated code for the loop through the dataset design.